File: test_archives/py_tests/batch4/test_function0.py
from functions.function0 import function0
import pytest

def test_function0_1():
    assert function0('A', 'B', 5, 10.5, True) == 'AB510.5'

def test_function0_2():
    assert function0('C', 'D', 3, 7.2, False) == 'CD37.2'

File: test_archives/cpp_tests/batch4/test_function0.cpp
#include "../catch.hpp"
#include "../functions/function0.cpp"

TEST_CASE( "Test function0_1", "[function0]" ) {
    REQUIRE( function0('A', 'B', 5, 10.5, true) == "AB510.5" );
}

TEST_CASE( "Test function0_2", "[function0]" ) {
    REQUIRE( function0('C', 'D', 3, 7.2, false) == "CD37.2" );
}